### PR TITLE
Improving inline quotes styling

### PR DESCRIPTION
**Pull request by rakatyal**
@riknoll, @dblotsky: Please review.

### CODE REVIEW

File: www/static/css-src/main.scss [hunk 1]
```diff
@@ -184,6 +184,12 @@ code.prettyprint {
     border: none !important;
 }
 
+/* improving inline quotes styling */
+code {
+    background-color: #c9dddf;
+    color: #000000;
```
Comment (riknoll) [inline www/static/css-src/main.scss]: The rest of the website uses `#333333` for text. Was this intentional?

Comment (rakatyal) [inline www/static/css-src/main.scss]: Yup. This one is a little darker shade than the original.

Comment (riknoll) [inline www/static/css-src/main.scss]: Cool

**Comment by dblotsky** [comment]
Should this be global, or within `_docs.scss`?

**Comment by rakatyal** [comment]
I wanted it to be for the blog as well.

**Comment by riknoll** [comment]
All the rest of our code formatting is global

**Comment by dblotsky** [comment]
Cool. LGTM then.

**Pull request closed without merging**